### PR TITLE
Transaction messages to sign

### PR DESCRIPTION
**Pull request by kilic**

### CODE REVIEW

File: contracts/FraudProof.sol [hunk 1]
```diff
@@ -198,31 +198,32 @@ contract FraudProofHelpers is FraudProofSetup {
...
-                    _tx
-                        .nonce,
-                    _tx
-                        .amount
-                )
-                    .ecrecovery(_tx.signature),
-            "Signature is incorrect"
-        );
-    }
+    // TODO: do we need this?
```
Comment (ChihChengLiang) [inline contracts/FraudProof.sol:201]: Remove commented

File: contracts/libs/RollupUtils.sol [hunk 2]
```diff
@@ -812,14 +815,6 @@ library RollupUtils {
         return _tx;
     }
 
-    function BurnExecutionSignBytes(uint256 txType, uint256 fromIndex)
```
Comment (ChihChengLiang) [inline contracts/libs/RollupUtils.sol:815]: Need to restore this for the time being for the tests to pass

File: contracts/libs/RollupUtils.sol [hunk 3]
```diff
@@ -690,14 +687,20 @@ library RollupUtils {
...
-        uint256 amount,
-        uint256 nonce
+        uint256 toIndex,
+        uint256 nonce,
+        uint256 amount
     ) public pure returns (bytes32) {
         return
             keccak256(
-                BytesFromBurnConsentNoStruct(txType, fromIndex, amount, nonce)
+                abi.encode(
```
Comment (vaibhavchellani) [inline contracts/libs/RollupUtils.sol:697]: Why call abi.encode instead of reusing the util?

File: contracts/libs/RollupUtils.sol [hunk 4]
```diff
@@ -531,20 +531,17 @@ library RollupUtils {
     }
 
     function getTxSignBytes(
-        uint256 txType,
```
Comment (vaibhavchellani) [inline contracts/libs/RollupUtils.sol:534]: Can we change this to TransferSignBytes or something like that?

File: contracts/libs/RollupUtils.sol [hunk 5]
```diff
@@ -692,12 +681,12 @@ library RollupUtils {
     function BurnConsentSignBytes(
         uint256 txType,
         uint256 fromIndex,
-        uint256 amount,
-        uint256 nonce
+        uint256 nonce,
+        uint256 amount
```
Comment (ChihChengLiang) [inline contracts/libs/RollupUtils.sol:685]: @vaibhavchellani want to confirm if we want this

Comment (vaibhavchellani) [reply contracts/libs/RollupUtils.sol:685]: Should be okay, ill update the client. But thanks for pointing out.

**Pull request merged**